contracts/ColonyFunding.sol

```diff
@@ -89,6 +89,8 @@ contract ColonyFunding is ColonyStorage, PatriciaTreeProofs {
   taskFinalized(_id)
   {
     Task storage task = tasks[_id];
+    require(task.roles[_role].user != address(0x0), "colony-funding-payout-null-recipient");
```

Comment:
This should be `assert` not `require` as per the discussion we had around it.

Reply:
Fair fair will update. Also hmm the coverage is above 94 this time... 😕